### ISSUE TITLE
Threads not shutting down correctly, force exiting

### ISSUE THREAD

**Issue by Flarkinater** [IzzelAliz/Arclight#449]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

arclight-1.16.5-1.0.22-SNAPSHOT-7c63a9e

### OS & Java versions

OS: Debian 10, Java: openjdk version "16.0.1" 2021-04-20

### Plugins and Mods

```raw
Mods:
 FastLeafDecay
 JEI

Plugins:
 AnimatedScoreboard
 ArclightPatcher
 AuctionHouse
 BKCommonLib
 CMI
 CMILib
 Center
 CoreProtect
 CustomHeads
 DeluxeMenus
 DiscordSRV
 DriveBackupV2
 FastAsyncWorldEdit (WorldEdit)
 GDHooks
 GriefDefender
 HolographicDisplays
 HolographicExtension
 IllegalStack
 ImageMaps
 InteractiveChat
 LightCleaner
 Minepacks
 Multiverse-Core
 Multiverse-Inventories
 Multiverse-NetherPortals
 NamelessMC
 NoSpectTab
 PlaceholderAPI
 Plan
 PlugMan
 ProtocolLib
 PunishGUI
 ServerListPlus
 SpecializedCrates
 TangledMaze
 TouchscreenHolograms
 Vault
 Votifier
 VotingPlugin
 LuckPerms
```

### Description

Whenever I stop the server with certain plugins installed, I get a WARN message:
`[Exit Thread/WARN] [Arclight/]: 20 threads not shutting down correctly, force exiting`

I tested all my plugins individually, and figured out that these plugins cause the warning to appear:
> AnimatedScoreboard.jar
CoreProtect
DiscordSRV
FastAsyncWorldEdit (with patcher-loader-1.0.0)
LuckPerms-Bukkit
Multiverse-Core
Multiverse-Inventories
Multiverse-NetherPortals
PlaceholderAPI
Plan
VotingPlugin

If any of these plugins are installed on their own (with no other plugins), the warning will show `1 thread not shutting down correctly`. The exception is VotingPlugin which shows `11 threads not shutting down correctly`

### Step to reproduce

1. Install the latest dev build of Arclight (arclight-1.16.5-1.0.22-SNAPSHOT-7c63a9e)
2. Install any of the plugins from the list in the description
3. Start the server
4. Stop the server
5. Check the console or latest.log. The last line is `XX threads not shutting down correctly, force exiting`

### Logs

[latest.log](https://github.com/IzzelAliz/Arclight/files/7867085/latest.log)

**Comment by HotoRas** [comment]
MVCore does not show that for me, are you using the latest version?
Also, if FastAsyncWorldEdit is a fork of the original WorldEdit, it may not support the hybrids(including Arclight).

Please demonstrate the version of plugins you've applied to server and I'll check the exact version with my latest build.

**Comment by IzzelAliz** [comment]
This is fixed in latest build.

**Comment by Flarkinater** [comment]
> This is fixed in latest build.

Can you explain what you mean by this? As far as I know I'm using the latest dev build for 1.16.5 (1.0.22-SNAPSHOT-7c63a9e)

**Comment by HotoRas** [comment]
> > This is fixed in latest build.
> 
> Can you explain what you mean by this? As far as I know I'm using the latest dev build for 1.16.5 (1.0.22-SNAPSHOT-7c63a9e)

Maybe the force exiting thread. They "fixed" with using that method, on 1.16.5-1.0.22.
But when to restart: the main thread doesn't stop but the new one opens, causes the log file to be failed writing.(Because of this I think it's not perfectly fixed though)

**Comment by zThana** [comment]
> Maybe the force exiting thread. They "fixed" with using that method, on 1.16.5-1.0.22.

I'm using 1.16.5-1.0.23 and am experiencing this issue.
